### ISSUE TITLE
Broken with MediaWiki 1.27, extension needs update

### ISSUE THREAD

**Issue by nuess0r** [vedmaka/Mediawiki-reCaptcha#6]
Hi
Today I updated my MediaWiki LTS installation from 1.23 to 1.27 and replaced ConfirmEdit with your extension because ConfirmEdit breaks Semantic MediaWiki.

Unfortunately your extension does not work with 1.27. It seems that the extension needs to be updated to work with the new AuthManager: [Manual:SessionManager_and_AuthManager#PreAuthenticationProvide](https://www.mediawiki.org/wiki/Manual:SessionManager_and_AuthManager#PreAuthenticationProvider)

The error I get in my webserver log looks like this:

`PHP Catchable fatal error:  Argument 1 passed to reCaptcha\\reCaptchaHooks::onUserCreateForm() must be an instance of UsercreateTemplate, instance of FakeAuthTemplate given in /mediawiki-extensions/Recaptcha/src/reCaptchaHooks.php on line 13,`

Thanks

**Comment by vedmaka** [comment]
That's correct, unfortunately reCaptcha is not compatible with latest AuthManager related changes of 1.27+ yet. Thanks for creating issue, maybe I'll have time to fix it in nearest time.

**Comment by rlorenzo** [comment]
Is this extension dead? We are creating a form in Mediawiki that needs to have recaptcha and would love to be able to use this extension.
